### PR TITLE
Support providing a custom OpenCV tarball

### PR DESCRIPTION
**Pull request by umireon**

### CODE REVIEW

File: cmake/BuildMyOpenCV.cmake [hunk 1]
```diff
@@ -1,5 +1,26 @@
 include(ExternalProject)
 
+set(CUSTOM_OPENCV_URL
+    ""
+    CACHE STRING "URL of a downloaded OpenCV tarball")
```
Comment (royshil) [inline cmake/BuildMyOpenCV.cmake]: Nit: you don't need "downloaded"
Just "URL of OpenCV tarball" can work ok

File: cmake/BuildMyOpenCV.cmake [hunk 2]
```diff
@@ -1,5 +1,26 @@
 include(ExternalProject)
 
+set(CUSTOM_OPENCV_URL
+    ""
```
Comment (royshil) [inline cmake/BuildMyOpenCV.cmake]: By the way if you know the default value (below, from GitHub) why not use it here as default value?

File: cmake/BuildMyOpenCV.cmake (hunk 1 above)
Comment (umireon) [reply cmake/BuildMyOpenCV.cmake]: Fixed!

File: cmake/BuildMyOpenCV.cmake (hunk 2 above)
Comment (umireon) [reply cmake/BuildMyOpenCV.cmake]: Fixed!

**Review by royshil — APPROVED** [review]
Looking good

**Pull request merged**